src/components/modal/Modal.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import './Modal.css';
+
+
+
+class Modal extends Component {
```

Comment:
Tämän komponentin voisi tehdä stateless-komponenttinä, jolloin this.props.show annetaan komponentille propsina.